packages/ozone/src/mod-service/index.ts

```diff
@@ -858,12 +857,14 @@ export class ModerationService {
       neg: !!l.neg,
     }))
     const { ref } = this.db.db.dynamic
+    await sql`notify ${ref(LabelChannel)}`.execute(this.db.db) // emitted transactionally
```

Comment:
Re: the comment—does this need to be emitted transactionally?  It seems like it probably doesn't need to be transactional, and I have a feeling it often runs outside of a transaction.